datascience-notebook/Dockerfile

```diff
@@ -23,4 +23,4 @@ RUN npm install jupyterlab-flake8 \
 
 # Fix conda install for NB_UID
 RUN chown -R $NB_UID /opt/conda \
- &&usermod -u $NB_UID $NB_USER
+ &&usermod -u $NB_UID $NB_USER
```

Comment:
Not sure the whitespaces were actually causing any build problem but for readability it would be best to add a space between the `&&` and `usermod`.

Reply:
Done